src/bandersnatch_storage_plugins/s3.py

```diff
@@ -123,6 +123,8 @@ class S3Storage(StoragePlugin):
     PATH_BACKEND = S3Path
     resource = None
     UPLOAD_TIME_METADATA_KEY = "uploaded-at"
+    CONFIG_PREFIX = "config_param_"
```

Comment:
```suggestion
    BOTO_CONFIG_PREFIX = "config_param_"
```
Maybe to make it clearer it's use

Reply:
Agreed -- that's more clear. Will update.